src/index.html

```diff
@@ -1,14 +1,34 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta charset="UTF-8" >
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" >
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.css" >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <section class="nav__section">
+        <picture class="header__logo">
+          <a href="#home"><img src="./images/logo.png" alt="mate-logo"></a>
+        </picture>
+        <nav class="nav__list">
+          <a class="is-active" href="#">apple</a>
+          <a href="#">samsung</a>
+          <a href="#">smartphones</a>
+          <a data-qa="hover" href="#">laptops & computers</a>
+          <a href="#">gadgets</a>
+          <a href="#">tablets</a>
+          <a href="#">photo</a>
+          <a href="#">video</a>
+        </nav>
```

Comment:
you should also use the ul and li tags here
<img width="436" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/70220667/5a1fbd3a-f9e6-498d-ae13-8ad311788722">